feconf.py

```diff
@@ -112,6 +112,10 @@
     TRAINING_JOB_STATUS_FAILED: [TRAINING_JOB_STATUS_NEW]
 }
 
+# The trivial interaction ids which don't need the true instance of
+# solicit_answer_details.
+CANNOT_SOLICIT_ANSWER_DETAILS = ['EndExploration', 'Continue']
```

Comment:
INTERACTION_IDS_WITHOUT_ANSWER_DETAILS

or

TRIVIAL_INTERACTION_IDS

Though actually I don't think you need a new constant here. Search for is_linear in the codebase, I think that represents the same concept and you can make use of that instead.

Reply:
I checked it, but only 'Continue' interaction has the linear property as true, but I want to have the 'EndExploration' too which has this property as false.

Reply:
OK, separate constant is fine then. Probably go with the first name, otherwise "trivial" is additional jargon.

Reply:
Done